03-client-api/03-nodejs.md

```diff
@@ -23,9 +23,9 @@ npm install grakn-client
 ```
 
 ## Quickstart
-First make sure, the [Grakn server](/docs/running-grakn/install-and-run#start-the-grakn-server) is running..
+First make sure that the [Grakn server](/docs/running-grakn/install-and-run#start-the-grakn-server) is running.
 
-In your source, require `grakn-client`.
+In your source, make `grakn-client` a requirement.
```

Comment:
`require` here is referring to the Javascript function `required()`. similarly, we use the keyword `import` for Python and Java.

Reply:
Out of curiosity, is it common to say "require `grakn-client`" in the Javascript world?

Because for example, maybe we can change it to "In your source, import `grakn-client`."

Reply:
@lolski `import` may only be used in ES6 modules which isn't supported by any native Javascript engine. normally (in most modern Javascript applications), Babel is used to convert ES6 syntax to CommonJS, but we can't assume that every developer uses it.

Reply:
Thanks Soroush ;)